modules/python/src2/gen2.py

```diff
@@ -30,7 +30,7 @@
 gen_template_simple_call_constructor_prelude = Template("""self = PyObject_NEW(pyopencv_${name}_t, &pyopencv_${name}_Type);
         if(self) """)
 
-gen_template_simple_call_constructor = Template("""self->v = ${cname}${args}""")
```

Comment:
Before such behavior made
`operator=()(<invalid object from python's malloc()>, Class(args))`
Now it directly constucts object on allocated data using placement new, no extra assigment is perfomed.